### ISSUE TITLE
brew tap does not work due to broken Aliases

### ISSUE THREAD

**Issue by BaconRanch** [OSGeo/homebrew-osgeo4mac#1384]
**Please answers the following**

I've... 
- I read
	- [X] the [reporting issues](https://osgeo.github.io/homebrew-osgeo4mac/docs/reporting-issues/)
	- [X] the [troubleshooting](https://osgeo.github.io/homebrew-osgeo4mac/docs/troubleshooting/)
- [X] I checked the documentation and found no answer
- [X] I tried to install after following the suggestions
- [X] I am running the latest version (`brew update` twice?)
- [X] I checked to make sure that this issue has not already been filed
- [ ] I uploaded logs: `brew gist-logs formula`

**Describe the bug**

Running `brew tap osgeo/osgeo4mac` on a fresh Homebrew install results in failure:
```
Updating Homebrew...
==> Tapping osgeo/osgeo4mac
Cloning into '/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac'...
remote: Enumerating objects: 192, done.
remote: Counting objects: 100% (192/192), done.
remote: Compressing objects: 100% (133/133), done.
remote: Total 12574 (delta 128), reused 119 (delta 59), pack-reused 12382
Receiving objects: 100% (12574/12574), 10.14 MiB | 19.47 MiB/s, done.
Resolving deltas: 100% (9138/9138), done.
Error: Non-file alias: /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Aliases/qgis-res
Error: Non-file alias: /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Aliases/pyqt-qt5
Error: Non-file alias: /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Aliases/qgis
Error: Non-file alias: /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Aliases/qgis-ltr
Error: Non-file alias: /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Aliases/sip-qt5
Error: Non-file alias: /usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Aliases/qgis3
Error: Cannot tap osgeo/osgeo4mac: invalid syntax in tap!
```
It looks like there are several Aliases that point to Formulae that do not exist.

**To Reproduce**

1. Run `brew tap osgeo/osgeo4mac`

**Expected behavior**

The tap should work.

**Additional context**

`brew doctor` output: 

```shell
Your system is ready to brew.
```

**Comment by alazarolop** [comment]
Thank you! It should be fixed with this commit 162feb93a93349e95e828e64fcbcfda0133b3035

**Comment by BaconRanch** [comment]
It works now!  Thank you!
